iotools.c

```diff
@@ -21,6 +21,7 @@
 #include <string.h>
 #include <unistd.h>
 #include "commands.h"
+#include "config.h"
```

Comment:
But I don't see how they are being used. Can we actually add the use of it, maybe in the `usage()` message or so?

Reply:
Yes.